src/tasks/platforms/common-platform-tasks.ts

```diff
@@ -0,0 +1,56 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import * as http from 'http'
+import * as Listr from 'listr'
+
+export namespace CommonPlatformTasks {
+  /**
+   * Checks whether cluster on which Che is being deployed accessible.
+   * Requires flags.domain to be set.
+   */
+  export function getPingClusterTask(flags: any): Listr.ListrTask {
+    return {
+      title: 'Check if cluster accessible',
+      skip: () => flags['skip-cluster-check'],
+      task: async (_ctx: any, task: any) => {
+        const domain: string = flags.domain
+        if (!domain) {
+          task.title = `${task.title}... "--domain" flag is not set. Cannot check cluster availability.'`
+          return
+        }
+
+        if (!await checkServer(domain, 80) || !await checkServer(domain, 443)) {
+          throw new Error(`Cannot reach cluster "${domain}". To skip this check add "--skip-cluster-check" flag.`)
```

Comment:
`neither  ${domain}:80 nor ${domain}:443`

Reply:
Indeed, thanks

Reply:
Instead of
`Cannot reach cluster "${domain}". To skip this check add "--skip-cluster-check" flag.`
it is better to log
`Cannot reach neither '${domain}:80' nor '${domain}:443'. It means the incorrect domain is used. Please, set the correct domain or bypass the check by using '--skip-domain-check' flag.`

Reply:
I do not think it is a good idea to print domain twice. I would like to avoid it.